Automattic/wp-calypso
# .gthub/CODEOWNERS: Fixes using `payments` team, which no longer exists

#### Proposed Changes

* The `Payments` team no longer exists, so the CODEOWNERS file needs to be updated to reference a team that does exist; `Shilling`

#### Testing Instructions



* Visually inspect [.github/CODEOWNERS](https://github.com/Automattic/wp-calypso/blob/4f17e425b3edf703c4ea8d95a0c2dbe0a5789aef/.github/CODEOWNERS#L35-L56) `# Payments` section of the file (lines 35 - 56)
* Check that all references to `@Automattic/payments` have been changed to `@Automattic/shilling` 

#### Pre-merge Checklist



- [ NA ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ NA ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ NA ] Have you checked for TypeScript, React or other console errors?
- [ NA ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ NA ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?



Related to #[66652](https://github.com/Automattic/wp-calypso/issues/66652)
